README.md

```diff
@@ -9,3 +9,20 @@ https://www.theodinproject.com/courses/ruby-programming/lessons/oop
 
 # Project Description
 Learning OOP with Ruby
+
+#Instructions
```

Comment:
**[REQUIRED]** The readme file should also tell how to start the game, try to imagine you are explaining how to initiate your game to someone with no idea of programming.